src/client/testing/create-availability.spec.ts

```diff
@@ -31,3 +31,53 @@ test('A user can navigate to the Create Availability flow from the site page', a
   await page.waitForURL('**/site/ABC02/create-availability');
   await expect(createAvailabilityPage.title).toBeVisible();
 });
+
+test('E2E Create single session of RSV availability', async ({ page }) => {
```

Comment:
Again it's a really minor thing and just my opinion, but I don't think you need to write `E2E` at the start of your test names. The other tests don't so it's a bit inconsistent, and since these are all end-to-end tests anyway it doesn't tell us anything we don't already know. 

If it's a pattern you want to introduce however, we should also add `E2E` to the start of all the other test names for consistency :)